automatic/kubernetes-kompose/update.ps1

```diff
@@ -15,24 +15,29 @@ function global:au_SearchReplace {
     ".\legal\VERIFICATION.txt" = @{
       "(?i)(^\s*location on\:?\s*)\<.*\>" = "`${1}<$($Latest.ReleaseURL)>"
       "(?i)(^\s*software.*)\<.*\>"        = "`${1}<$($Latest.URL64)>"
-      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType32)"
+      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType64)"
       "(?i)(^\s*checksum\:).*"            = "`${1} $($Latest.Checksum64)"
     }
+
+    "$($Latest.PackageName).nuspec" = @{
+      "(\<copyright\>.*?)\d{4}" = "`${1}$($Latest.ReleaseYear)"
```

Comment:
Can you guarantee that this will guarantee that the copyright will be equal to the copyright by the Software Authors?

If not, then this should be a manual process.